tools/SetupFlow/DevHome.SetupFlow/ViewModels/ReviewViewModel.cs

```diff
@@ -138,5 +143,44 @@ private async Task OnSetUpAsync()
         {
             _log.Error($"Failed to initialize elevated process.", e);
         }
-   }
+    }
+
+    [RelayCommand(CanExecute = nameof(HasDSCTasksToDownload))]
+    private async Task DownloadConfigurationAsync()
+    {
+        try
+        {
+            // Supported save file types
+            var wingetFileType = ".winget";
+            var dscFileType = ".dsc.yaml";
+            var filterTypes = new List<(string, string)>()
+            {
+                ($"*{wingetFileType}", StringResource.GetLocalized(StringResourceKey.FilePickerSingleFileTypeOption, "YAML")),
+                ($"*{dscFileType}", StringResource.GetLocalized(StringResourceKey.FilePickerSingleFileTypeOption, "YAML")),
+            };
+
+            // Show the save file dialog
+            var result = _mainWindow.SaveFileDialog(Log.Logger, filterTypes.ToArray());

```

Comment:
Create issue to consider enhancing the file dialog experience. Maybe appending the file type should be an option that can be passed to the save file dialog instead, so other users can benefit from it,

Reply:
https://github.com/microsoft/devhome/pull/2492